3-ML-Basic.sh

```diff
@@ -31,6 +31,22 @@ run_and_echo () {
 }
 
 
+if [[ (-n $(lspci | grep -i nvidia)) && (! ( -d "/usr/local/cuda" ) ) ]]; then
+    echo "Installing the latest cuda"
+    cuda_instr_block=$(wget -q -O - 'https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=1804&target_type=runfilelocal' | grep wget | head -n 1)
```

Comment:
Should it be different for `16.04` and `18.04`?

Reply:
Well, it's actually the same for both, so doesn't matter I guess

Reply:
Yeah...but the URL can be made version agnostic nonetheless, good suggestion